account_asset_management/models/account_asset.py

```diff
@@ -593,8 +593,10 @@ def group_lines(x, y):
                             'name': name,
                             'line_date': line['date'],
                             'line_days': line['days'],
-                            'init_entry': entry['init'],
+                            'init_entry': False,
                         }
+                        if line['days'] == 0:
```

Comment:
This is not correct at all: we should mark as init entry those before the locking date, not only the first. What is not correct is to mark all the year entries, no matter the locking exact date.